apps/meteor/app/lib/server/functions/sendMessage.ts

```diff
@@ -283,6 +283,8 @@ export const sendMessage = async function (user: any, message: any, room: any, u
 		message._id = insertedId;
 	}
 
+	console.log('Inserted message', message);
```

Comment:
```suggestion
```

Reply:
I did notice that my test log was missing from on my other branch 🤣